CMR/python/README.md

```diff
@@ -0,0 +1,49 @@
+# CMR Python Wrapper
+
+A wrapper for https://github.com/nasa/Common-Metadata-Repository/ in python
+
+## Assumptions
+
+1. Python 3.2 or better.
+2. Can access [cmr.earthdata.nasa.gov][cmr]
+3. Optional: an account on [urs.earthdata.nasa.gov][urs]
+
+## Styles
+
+1. Coding Styles are defined as [PEP 8][pep8].
+2. This project uses the [editorconfig.org][econfig] file .editorconfig 
+3. Use: `pylint *` to check code
+
+## Usage
+
+### Getting a token
+
+There are three ways to store a password for use in scripts. Plain text in the script (not recommended), in a file, but still clear text, or if your a mac or compatible system, you can use the password manager which on Mac OS X will come from Keychain. These three methods are selected by passing in a lambda function to the token() function
```

Comment:
Grammar: [if your a...] -> [if you're using a...], missing period at the end of the sentence

Reply:
English is my second language, I'll make that 'you are'.